flang/test/Lower/OpenMP/omp-wsloop-collapse.f90

```diff
@@ -0,0 +1,240 @@
+! LLVMIRDialect:           %[[VAL_10:.*]] = llvm.alloca %[[VAL_9]] x i32 {bindc_name = "b", in_type = i32, operand_segment_sizes = dense<0> : vector<2xi32>, uniq_name = "_QEb"} : (i64) -> !llvm.ptr<i32>
+! LLVMIRDialect:           %[[VAL_11:.*]] = llvm.mlir.constant(1 : i64) : i64
+! LLVMIRDialect:           %[[VAL_12:.*]] = llvm.alloca %[[VAL_11]] x i32 {bindc_name = "c", in_type = i32, operand_segment_sizes = dense<0> : vector<2xi32>, uniq_name = "_QEc"} : (i64) -> !llvm.ptr<i32>
+! LLVMIRDialect:           %[[VAL_13:.*]] = llvm.mlir.constant(1 : i64) : i64
+! LLVMIRDialect:           %[[VAL_14:.*]] = llvm.alloca %[[VAL_13]] x i32 {bindc_name = "i", in_type = i32, operand_segment_sizes = dense<0> : vector<2xi32>, uniq_name = "_QEi"} : (i64) -> !llvm.ptr<i32>
+! LLVMIRDialect:           %[[VAL_15:.*]] = llvm.mlir.constant(1 : i64) : i64
+! LLVMIRDialect:           %[[VAL_16:.*]] = llvm.alloca %[[VAL_15]] x i32 {bindc_name = "j", in_type = i32, operand_segment_sizes = dense<0> : vector<2xi32>, uniq_name = "_QEj"} : (i64) -> !llvm.ptr<i32>
+! LLVMIRDialect:           %[[VAL_17:.*]] = llvm.mlir.constant(1 : i64) : i64
+! LLVMIRDialect:           %[[VAL_18:.*]] = llvm.alloca %[[VAL_17]] x i32 {bindc_name = "k", in_type = i32, operand_segment_sizes = dense<0> : vector<2xi32>, uniq_name = "_QEk"} : (i64) -> !llvm.ptr<i32>
+! LLVMIRDialect:           %[[VAL_19:.*]] = llvm.mlir.constant(1 : i64) : i64
+! LLVMIRDialect:           %[[VAL_20:.*]] = llvm.alloca %[[VAL_19]] x i32 {bindc_name = "x", in_type = i32, operand_segment_sizes = dense<0> : vector<2xi32>, uniq_name = "_QEx"} : (i64) -> !llvm.ptr<i32>
+! LLVMIRDialect:           llvm.store %[[VAL_0]], %[[VAL_8]] : !llvm.ptr<i32>
+! LLVMIRDialect:           llvm.store %[[VAL_1]], %[[VAL_10]] : !llvm.ptr<i32>
+! LLVMIRDialect:           llvm.store %[[VAL_2]], %[[VAL_12]] : !llvm.ptr<i32>
+! LLVMIRDialect:           llvm.store %[[VAL_3]], %[[VAL_20]] : !llvm.ptr<i32>
+! LLVMIR:         %[[VAL_0:.*]] = alloca i32, i64 1, align 4, !dbg !7
+! LLVMIR:         %[[VAL_1:.*]] = alloca i32, i64 1, align 4, !dbg !9
+! LLVMIR:         %[[VAL_2:.*]] = alloca i32, i64 1, align 4, !dbg !10
+! LLVMIR:         %[[VAL_3:.*]] = alloca i32, i64 1, align 4, !dbg !11
+! LLVMIR:         %[[VAL_4:.*]] = alloca i32, i64 1, align 4, !dbg !12
+! LLVMIR:         %[[VAL_5:.*]] = alloca i32, i64 1, align 4, !dbg !13
+! LLVMIR:         %[[VAL_6:.*]] = alloca i32, i64 1, align 4, !dbg !14
+! LLVMIR:         store i32 3, i32* %[[VAL_0]], align 4, !dbg !15
+! LLVMIR:         store i32 2, i32* %[[VAL_1]], align 4, !dbg !16
+! LLVMIR:         store i32 5, i32* %[[VAL_2]], align 4, !dbg !17
+! LLVMIR:         store i32 0, i32* %[[VAL_6]], align 4, !dbg !18
+! LLVMIR:         %[[VAL_7:.*]] = call i32 @__kmpc_global_thread_num(%[[VAL_8:.*]]* @1), !dbg !19
+! LLVMIR:         br label %[[VAL_9:.*]]
+
+  !$omp parallel do collapse(3)
+! FIRDialect-LABEL:   omp.parallel {
+! LLVMIRDialect:           omp.parallel {
```

Comment:
Nit: FIR check uses label while llvm dialect check does not.

Reply:
Neither needs label, so removed.